### DISCUSSION TITLE
Jumper T-Pro V2 screen flipped with non-Jumper provided firmware

### DISCUSSION THREAD

**Discussion by peeterpiik** [EdgeTX/edgetx#3656] [Q&A] [answered]
Hi, I recently received the Jumper T-Pro V2 I purchased and one of the updates compared to the original version is that according to Jumper the transmitter now has an OLED screen which has quite nice looking white text on a black background however with all other firmwares than the 2.9.0-selfbuild version of EdgeTX that came on the radio from the factory (backed this up in the beginning) the image on the screen is flipped.

![IMG_1613](https://github.com/EdgeTX/edgetx/assets/68543654/13a461d7-7ceb-4e02-b881-260a23d59b0f)

So I'm wondering what my options are -  should I submit a request for a custom build of firmware, should I submit a bug report or pull request instead or would I need to learn how to compile custom firmware myself?

--edit--
Before submitting this post I searched for "T-Pro screen" here and got no results but now that I joined the Discord server I saw that this is a known issue which will be fixed with separate V2 firmware.

**Comment by gagarinlg** [comment, marked as answer]
See #3570
